### ISSUE TITLE
Utilities: Add support for k, M, G suffixes for dd

### ISSUE THREAD

**Issue by gunnarbeutner** [SerenityOS/serenity#6889]
This should be fairly simple to implement but would make using `dd` a little easier.

```
courage:~ $ dd if=/dev/zero of=/dev/null bs=1k count=1
Invalid size-value: 1k
```

**Comment by MaxWipfli** [comment]
This was implemented in #6906.

**Comment by gunnarbeutner** [comment]
Indeed it was, well done. :partying_face:
